analysis/import/GitLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/gitlogparser/GitLogParser.kt

```diff
@@ -144,15 +144,14 @@ class SCMLogParserV2(
     ): Project {
         val namesInProject = readFileNameListFile(pathToNameTree)
         val encoding = guessEncoding(pathToLog) ?: "UTF-8"
-        error.println("ATTENTION: This is an EXPERIMENTAL PARSER, use at your own risk!")
         if (!silent) error.println("Assumed encoding $encoding")
         val lines: Stream<String> = Files.lines(pathToLog.toPath(), Charset.forName(encoding))
         val projectConverter = ProjectConverter(containsAuthors)
         val logSizeInByte = file!!.length()
-        return SCMLogProjectCreator(parserStrategy, metricsFactory, projectConverter, logSizeInByte, silent).parse(
+        return GitLogProjectCreator(parserStrategy, metricsFactory, projectConverter, logSizeInByte, silent).parse(
             lines,
             namesInProject
-        )
+                                                                                                                  )
```

Comment:
It seems that the commit hooks are not activated because several whitespaces were added.